web/src/hooks/use-enriched-content/use-enriched-content.tsx

```diff
@@ -0,0 +1,35 @@
+import UserTag from '@/components/user-tag';
+import type { ReactNode } from 'react'
+import type { User } from '@/feedbackr-api/v1/schemas'
+
+export default function useEnrichedContent(content: string, users: User[]) {
+  const taggedUsers = content.match(/\@(\S+)/g);
+  if (!taggedUsers) return content;
+
+  const enrichedContent: ReactNode[] = [];
+  let startIndex = 0;
+
+  taggedUsers.forEach((taggedUser) => {
+    const index = content.indexOf(taggedUser, startIndex);
+    if (index !== -1) enrichedContent.push(content.substring(startIndex, index));
+    
+    const username = taggedUser.substring(1);
+
+    const user = users.find((user) => user.username === username)
```

Comment:
```suggestion
    const user = users.find((user) => user.username.toLowCase() === username.toLowCase())
```

What do you think? In order to make it case insensitive, because right now `@Rick-Grimes` doesn't return `@rick-grimes`

with the changes, `@Rick-Grimes`  will return `@rick-grimes`

Reply:
almost there. It's `toLowerCase`, not `toLowCase`

![3be2367401f51939946a01098301af1fcafb2912_hq](https://github.com/mapra99/feedbackr/assets/27536621/b8ad89d3-c467-4c7f-833e-f8bad225050a)

Reply:
hahahahahhahahahhahahahahhahahaha `lower` hahahahhahahahhaha